src/main/java/software/amazon/documentdb/jdbc/common/utilities/SqlError.java

```diff
@@ -16,6 +16,7 @@
 
 package software.amazon.documentdb.jdbc.common.utilities;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
Can we avoid including it? I believe it also can cause confusion if it is a GPL license.

Reply:
The alternative is to add it to the "ignore" file - which I think separates the suppression from the configuration. 

But open.

Reply:
Removed the annotation from the main source code. Migrated to use the spotbugs-exclude.xml configuration file. Note: the library did not make it into the UBER jar.

Reply:
@alexey-temnikov Resolved the CRLF_INJECTION. 
Let me know if you have any other concerns.